samples/data-ingestion/README.md

```diff
@@ -0,0 +1,108 @@
+---
```

Comment:
These arent different samples, they're just guides to different features, shouldnt need separate sample listings?